testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/security/SimpleSessionCookieIT.java

```diff
@@ -32,17 +54,144 @@ public SimpleSessionCookieIT(final String nexusBundleCoordinates) {
     super(nexusBundleCoordinates);
   }
 
+  @Override
+  protected NexusBundleConfiguration configureNexus(NexusBundleConfiguration configuration) {
+    // help verify that a custom context path sets the cookie path value correctly
+    return super.configureNexus(configuration).setContextPath("/customcontextpath");
+  }
+
+  @Test
+  public void defaultSessionCookieSpecUsingHttps() throws Exception {
+    exerciseCookieSpec(nexus().getSecureUrl());
```

Comment:
This should also check that the cookie has the `isSecure` flag set as expected while exercising the cookie tests, as otherwise this IT will still pass even when the new cookie filter is not taking effect.